### DISCUSSION TITLE
Problemas com a criação de uma conta

### DISCUSSION THREAD

**Discussion by MigsRS123** [vercel/community#3919] [Account Issue]
### Question

Tentei criar uma conta com o github, mas dizia que a conta estava bloqueada. Escrevi por aqui esperando por alguma ajuda, depois de um tempo me enviaram dois e-mails, os quais eram um pedido de desculpa e outro dizendo que o problema estava resolvido, tentei novamente entrar mas ainda diz que a conta está bloqueada.

**Comment by wincklerhigher** [comment]
Estou com o mesmo problema do colega acima, porém eu não recebi e-mails dizendo que o problema esteja resolvido. Apenas um e-mail para confirmação da conta. 
Com o Github dá a seguinte mensagem: "This user account is blocked."
E com e-mail: "we are unable to complete your signup. Please read this guide for further details."
